examples/00_tutorials/plot_python_101.py

```diff
@@ -3,17 +3,20 @@
 =======================================
 """
 
-try:
-    import matplotlib.pyplot as plt
-except ImportError:
+from nilearn._utils.helpers import is_matplotlib_installed
+
+if not is_matplotlib_installed():
     raise RuntimeError("This script needs the matplotlib library")
```

Comment:
This repeats in many places. An option would be to implement it as a function in `helpers`.

Reply:
yeah that's a good point: let me refactor this.

Reply:
done